connector/connectorRunner.go

```diff
@@ -60,10 +62,31 @@ func (cr *connectorRunner) Run() error {
 		return err
 	}
 
+	var (
+		server *factory.GRPCServer
+		writer process.Writer
+	)
+	if cr.config.GRPC.Enable {
+		writer = &fakeWriter{}
```

Comment:
this logic should be better put in factory package

Reply:
done